server/routes/ad.ts

```diff
@@ -963,8 +963,9 @@ export default class AdService {
 
       const detectorResult: AnomalyResult[] = [];
       const featureResult: { [key: string]: FeatureResult[] } = {};
-
+      
```

Comment:
nit: you might have done this already but you should run the linter on everything, could be in a different PR if you have another. https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/DEVELOPER_GUIDE.md#formatting

Reply:
yeah I didn't know this. Will run it in the next pr